### PR TITLE
issue-1588: Support RPC_GET_VOLUME_STATS capability in csi driver

### PR DESCRIPTION
**Pull request by antonmyagkov**
issue: #1588

CSI driver provides RPC_GET_VOLUME_STATS capability only in podMode when volume is mounted to the pod as a local folder.

### CODE REVIEW

File: cloud/blockstore/tools/csi_driver/internal/driver/node_test.go
Comment (jkuradobery) [inline cloud/blockstore/tools/csi_driver/internal/driver/node_test.go:444]: Since the csi driver is built using the same ya.make as disk-manager, consider using testify/require.
This line would look: require.NoError(t, err)

File: cloud/blockstore/tools/csi_driver/internal/driver/node.go
Comment (jkuradobery) [inline cloud/blockstore/tools/csi_driver/internal/driver/node.go:682]: Missing line break

File: cloud/blockstore/tools/csi_driver/internal/driver/node_test.go
Comment (jkuradobery) [inline cloud/blockstore/tools/csi_driver/internal/driver/node_test.go:431]: The comments are not needed. Either use variables or remove them completely (I would remove them)

File: cloud/blockstore/tools/csi_driver/internal/driver/node_test.go
Comment (jkuradobery) [inline cloud/blockstore/tools/csi_driver/internal/driver/node_test.go:468]: It seems that you are just checking for the invariant. It would be nice to create some files and then inspect actual inodes count and sizes.

**Review by jkuradobery — APPROVED** [review]

**Review by jkuradobery — CHANGES_REQUESTED** [review]
Sorry, once again accidentally approved after leaving discussions

File: cloud/blockstore/tools/csi_driver/internal/driver/node.go
Comment (qkrorlqr) [inline cloud/blockstore/tools/csi_driver/internal/driver/node.go:55]: мм, наоборот же - в vmMode мы НЕ поддерживаем GET_VOLUME_STATS, а в случае не-vmMode (то есть монтирование папки в pod) - поддерживаем

File: cloud/blockstore/tools/csi_driver/internal/driver/node.go
Comment (antonmyagkov) [reply cloud/blockstore/tools/csi_driver/internal/driver/node.go]: fixed

File: cloud/blockstore/tools/csi_driver/internal/driver/node_test.go
Comment (antonmyagkov) [reply cloud/blockstore/tools/csi_driver/internal/driver/node_test.go]: removed

File: cloud/blockstore/tools/csi_driver/internal/driver/node_test.go
Comment (antonmyagkov) [reply cloud/blockstore/tools/csi_driver/internal/driver/node_test.go]: removed

File: cloud/blockstore/tools/csi_driver/internal/driver/node.go
Comment (antonmyagkov) [reply cloud/blockstore/tools/csi_driver/internal/driver/node.go]: Fixed

File: cloud/blockstore/tools/csi_driver/internal/driver/node_test.go
Comment (antonmyagkov) [reply cloud/blockstore/tools/csi_driver/internal/driver/node_test.go:472]: I'm not sure that it's possible because in fact we don't mount volume in this test and overall numbers of inodes/bytes depend on the state of vm filesystem. 

By the way I added another assert to check that total number of inodes/bytes are not zero.

File: cloud/blockstore/tools/csi_driver/internal/driver/node.go
Comment (qkrorlqr) [inline cloud/blockstore/tools/csi_driver/internal/driver/node.go:686]: only in podMode

File: cloud/blockstore/tools/csi_driver/internal/driver/node_test.go
Comment (antonmyagkov) [reply cloud/blockstore/tools/csi_driver/internal/driver/node_test.go]: done

File: cloud/blockstore/tools/csi_driver/internal/driver/node.go
Comment (antonmyagkov) [reply cloud/blockstore/tools/csi_driver/internal/driver/node.go]: поменял на: NodeGetVolumeStats is not supported in vmMode

File: cloud/blockstore/tools/csi_driver/internal/driver/node_test.go
Comment (jkuradobery) [inline cloud/blockstore/tools/csi_driver/internal/driver/node_test.go:509]: require.NoError

Comment (antonmyagkov) [reply cloud/blockstore/tools/csi_driver/internal/driver/node_test.go]: Fixed

**Review by jkuradobery — APPROVED** [review]

**Review by drbasic — APPROVED** [review]

**Pull request merged**
